### ISSUE TITLE
noob question 

### ISSUE THREAD

**Issue by futuremoneybtc** [JeanLucPons/Kangaroo#27]
Is there any way to run those types of scripts through mining machines any information regarding my question is appreciated.. thank you

**Comment by WanderingPhilosopher** [comment]
When you say "mining machines" what do you mean?

**Comment by futuremoneybtc** [comment]
like 50 TH/S speed

**Comment by CatfishCrypt** [comment]
Like an ASIC? I don't think so, since those are built to do just one thing...

**Comment by djarumlights** [comment]
Correct. This is not the same operation that goes into mining. ASICs are made to do one specific set of math and that set of math only. (Application _Specific_ Integrated Circuit)

Someone might could get crazy with an FPGA, but that takes a WHOLE different set of programming skills.

**Comment by dem10** [comment]
Yes, FPGA are more powerful devices than a video card, but will have to write a new program for it.
